script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -494,128 +605,147 @@ The fact of the matter is that I'm awfully calm right now.
 [sha:48cef7a445c8d7b5bd457ea66e9c9ce8832afbf7]{
 -- Page 144, Offset 35653.
 -- その証拠に、鉄柵の周りには死者たちが集まってきているコトも把握している。
-As a testament to that, I can even tell that the Dead are gathering around the iron fence.
+So calm that it hasn't escaped my notice that the Dead are starting to gather around the iron fence.
 }
 [sha:53c5bc86f5044523755f38f3c161120e499c7fa6]{
 -- Page 144, Offset 35654.
 -- だから、頭痛を押し殺して最後の質問だってできる。
-That's why I'm able to stifle my headache and ask my final question.
+// だから because he's calm or because the Dead are getting kinda close?
+And so, I'm able to stifle my headache and ask my final question.
 }
 [sha:80a6bf184e170e2cc45cf992c9130694afb3619a]{
 -- Page 145, Offset 35655.
 -- 「ノエル先生。これが貴女との最後の会話でしょうけど」
 // boys in the dover moment
-"Noel-sensei. This might be the <final chance|s a i g o> I get to speak with you."
+"Noel-sensei.
+ This will probably be my last conversation with you."
 }
 [sha:db916fbf48b576bc507fbe6baf05a5d7f63bb725]{
 -- Page 145, Offset 35656.
 -- 「最期、の間違いよ志貴クン。日本語は正しくね？」
-"Don't you mean your <'final moment'|s a i g o>, Shiki-kun? Isn't your Japanese a bit off?"
+"Did you misspeak, Shiki-kun?
+ I think you meant to say that these will be your last %{i}words%{/i}."
 }
 [sha:50794bd0bbf2d02ce95f960425b548ed2ed97807]{
 -- Page 145, Offset 35657.
 -- 「……むざむざ殺される気はありません。それでも、貴女が俺だけを殺すというのなら、筋は通っています」
-"...I don't plan on going down without a fight. Still, I would understand if you were only intent on killing me."
+"I don't plan on going down without a fight.
+ Still... I can understand why you'd want to kill %{i}me%{/i}, but..."
```

Comment:
I feel like it's more along these lines.
It's less that he understands, and more that he feels like "yeah, that's fair and just"
>Still... if the only person you wanted to kill was me, you'd have all the right in the world.